android/app/build.gradle

```diff
@@ -109,10 +109,12 @@ android {
     }
     signingConfigs {
         release {
+            /*
             storeFile file(AGRITRADER_DEPLOY_UNSTABLE_STORE_FILE)
             storePassword AGRITRADER_DEPLOY_UNSTABLE_STORE_PASSWORD
             keyAlias AGRITRADER_DEPLOY_UNSTABLE_KEY_ALIAS
             keyPassword AGRITRADER_DEPLOY_UNSTABLE_KEY_PASSWORD
+            */
```

Comment:
I have not kept up to date with the changes occurring regarding this, I thought James' PR fixed this already?

Reply:
I reverted this so the APK would build. If people went back in history, they should have something they should've built.  But my mistake, i should have recommented them in a new commit so it woudln't break the build again. But we do need to find a longer term solution to this.

Reply:
This was just something I had to do when I pulled off the milestone branch. I was unaware of James' fix for it. If I had to change it back, I'm sorry

Reply:
Don't worry I don't think you did anything wrong, I have no idea what's been going on around this and why it hasn't affected me yet haha

Reply:
Haha okay perfect. Thanks for the heads up still